pkgs/top-level/all-packages.nix

```diff
@@ -9406,6 +9406,8 @@ let
     inherit (vamp) vampSDK;
   };
 
+  mjpg_streamer = callPackage ../applications/video/mjpg-streamer { };
```

Comment:
I saw some of the gnome packages used "-" in the filename, but "_" as the
attribute name, so I went with that. `mjpg-streamer` is indeed more
consistent in this case, so I've changed it.

On Tue, Jan 20, 2015 at 10:55 AM, lethalman <notifications@github.com>
wrote:

> In pkgs/top-level/all-packages.nix
> <https://github.com/NixOS/nixpkgs/pull/5532#discussion-diff-23211911>:
>
> > @@ -9406,6 +9406,8 @@ let
> >      inherit (vamp) vampSDK;
> >    };
> >
> > +  mjpg_streamer = callPackage ../applications/video/mjpg-streamer { };
>
> Last thing, what about mjpg-streamer here as attribute name?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/NixOS/nixpkgs/pull/5532/files#r23211911>.
>